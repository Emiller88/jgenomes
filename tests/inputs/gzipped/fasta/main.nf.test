nextflow_pipeline {

    name "gzipped fasta"
    script "../../../../main.nf"
    tag "input"
    tag "fasta"

    test("Should run with a gzipped fasta") {
        when {
            params {
                input = "${projectDir}/tests/inputs/gzipped/fasta/input.yml"
                outdir = "$outputDir"
            }
        }

        then {
            assertAll(
                { assert workflow.success },
                { assert snapshot(UTILS.removeNextflowVersion("$outputDir/pipeline_info/nf_core_references_software_mqc_versions.yml")).match("software_versions") },
                // { assert snapshot(
                //     // FIXME workflow.trace.tasks().size(),
                // ).match("output_files")
                // }
            )
        }
    }
}
