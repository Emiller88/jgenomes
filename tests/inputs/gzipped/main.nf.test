nextflow_pipeline {

    name "gzipped inputs"
    script "../../../main.nf"
    tag "input"

    test("Should run with a gzipped gtf") {
        tag "gtf"

        when {
            params {
                input = "${projectDir}/tests/inputs/gzipped/gzip_gtf.yml"
                outdir = "$outputDir"
            }
        }

        then {
            assertAll(
                { assert workflow.success },
                { assert snapshot(UTILS.removeNextflowVersion("$outputDir/pipeline_info/nf_core_references_software_mqc_versions.yml")).match("software_versions") },
                // { assert snapshot(
                //     // FIXME workflow.trace.tasks().size(),
                // ).match("output_files")
                // }
            )
        }
    }
}
