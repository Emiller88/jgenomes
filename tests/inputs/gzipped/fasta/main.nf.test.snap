{
    "software_versions": {
        "content": [
            "{Workflow={nf-core/references=v1.0dev}}"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-09-11T14:13:23.171953655"
    }
}